vast.py

```diff
@@ -3599,7 +3598,83 @@ def transfer__credit(args: argparse.Namespace):
         print(r.text);
         print("failed with error {r.status_code}".format(**locals()));
 
+@parser.command(
```

Comment:
commands should be sorted in the file to show up in correct sorted order

Reply:
Whoops, fixed